### DISCUSSION TITLE
Add syntax highlighting for Python + HTML in SublimeText

### DISCUSSION THREAD

**Discussion by marimeireles** [pyscript/pyscript#1177] [Proposals]
It'd be awesome to set up double syntax highlighting for Sublime text: Python + HTML in the same file.
It seems like [1](https://stackoverflow.com/questions/47757055/multiple-syntax-highlighting-in-sublimetext-3-with-sublime-syntax-file-type), [docs](http://www.sublimetext.com/docs/syntax.html), it's possible and well documented.

**Comment by nmstoker** [comment]
I see there's an extension that does this in VS Code now too: https://marketplace.visualstudio.com/items?itemName=HardeepSingh.pyscript

**Comment by UltraInstinct05** [comment]
Here is a ST package that provides syntax highlighting for PyScript https://github.com/Sublime-Instincts/PyScript

**Comment by nmstoker** [comment]
Thanks for this!
I'll give it a go when https://github.com/Sublime-Instincts/PyScript/issues/1 is resolved

**Comment by UltraInstinct05** [comment]
Note that the README has easy alternate installation instructions for it, if you want to give it a test run.

**Comment by nmstoker** [comment]
Understood. You can't miss it 🙂

**Comment by imbev** [comment]
Has anyone found a corresponding plugin for vim?
